fy22_cecil_b_moore/race_breakdown.sql

```diff
@@ -0,0 +1,9 @@
+select q2, count(q2) from longform_online lo 
+group by q2
+
+select q2, count(q2) from  in_person_survey_response_data_entry ipsrde 
+group by q2
+
+select q2, count(q2) from longform_joined lj 
```

Comment:
I may have missed this in your commits, but what's the difference between `longform_online` and `longform_joined`?